mimic/core.py

```diff
@@ -35,6 +36,7 @@ def __init__(self, clock, apis):
         """
         self._uuid_to_api = {}
         self.sessions = SessionStore(clock)
+        self.message_store = MessageStore()
```

Comment:
Non-blocking - I just wanted to put this here and make a reference to #108, so it's kept in mind when we look at that issue again.  There should probably be standard place for plugins that do not make use of sessions to store their own data too, so not everyone has to add another item to core.  (e.g. maybe core could create all the storage objects by plugin).